### ISSUE TITLE
Vendored `typeshed` is totally obsolete

### ISSUE THREAD

**Issue by daskol** [davidhalter/jedi#2009]
At the moment, vendored `typeshed` package knows only Python up to 3.9 whilst the trunk has support for Python 3.12 and many third party stubs.

**Comment by davidhalter** [comment]
Yes, I know, but it's kind of hard to update it. But definitely something we should look into changing!
